### DISCUSSION TITLE
web3 IPFS 패키지에 관하여,

### DISCUSSION THREAD

**Discussion by EthanKIMHG** [codestates-beb/BEB-06-Angry-Cactus#27] [Q&A]
클라이언트 사이드에서 IPFS에서 올리려면 여러가지 방법이 있는것 같습니다.

NFT.Stroage 패키지 매니저를 사용해도 될거같고,

web3.storage 를 사용해도 될거같습니다.

승훈님은 어떤 패키지를 사용하실 예정이신가요?

**Comment by sinsu1004** [comment]
IPFS에 접근할 수 있는 모듈이 많지만 일반 파일을 업로드 할 때는 Web3.Storage를 사용하고, NFT를 발행할 때 필요한 tokenURI는 NFT Storage를 많이 사용한다고 합니다. 
NFT.Stroage 가 좋을 것 같습니다.
